diff --git a/data/resources/ui/message_item.blp b/data/resources/ui/message_item.blp
index 5d94128..36c540a 100644
--- a/data/resources/ui/message_item.blp
+++ b/data/resources/ui/message_item.blp
@@ -256,7 +256,6 @@ template $FlMessageItem: $ContextMenuBin {
             hexpand: true;
             halign: start;
             label: _("Sent a message type that is currently not supported by Flare");
-            attributes: "family italic";
             visible: bind template.message as <$FlMessage>.is-unsupported;
           }
 
@@ -272,7 +271,6 @@ template $FlMessageItem: $ContextMenuBin {
             hexpand: true;
             halign: start;
             label: _("Message was deleted");
-            attributes: "family italic";
             visible: bind template.message as <$FlMessage>.is-deleted;
           }
 
